nurse/api.py

```diff
@@ -1,8 +1,8 @@
-import inspect
+from nurse.exceptions import DependencyError
 from .service_catalog import ServiceCatalog
 
 
-def inject(user_class):
+def inject(*decorator_args):
     """
     A decorator that injects dependencies into every instances of a user-defined class.
```

Comment:
Je suggère de mettre à jour ce commentaire: par exemple `A decorator that injects dependencies into every instances of a user-defined class or method`